examples/react-router/package.json

```diff
@@ -10,8 +10,15 @@
     "meteor-node-stubs": "~0.2.0",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "react-komposer": "^2.0.0",
     "react-router": "^3.0.2",
-    "@okgrow/auto-analytics": "^1.0.2"
+    "@okgrow/auto-analytics": "^1.0.0"
```

Comment:
Should this be 2.0.0?

Reply:
@ccuilla Yes, I left it out though as we `.gitignore` the `node_modules` if you try to `meteor npm install` it will error out. Will bump it to `^2.0.0` after we get the all clear.

Temp instructions to get the Meteor app running.
To get the example running I have been running `meteor npm install`, and copying over the `dist/index.umd.js` into `imports` and then replacing line 9 in`main.jsx` with `import { initAnalytics } from '../imports/index.umd';`.